autotest/pyscripts/test_gdal_utils.py

```diff
@@ -174,6 +174,27 @@ def test_utils_color_files():
             assert cp1.pal[k] == v
 
 
+def test_utils_color_files_nv():
+    """ test color palettes with and without nv """
+    items = [
```

Comment:
you could use @pytest.mark.parametrize("name,ndv",[['color_paletted_red_green_0-255.txt', None], ['color_paletted_red_green_0-1-nv.txt'', 0]])

Reply:
nice, thanks!